libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBTableColumn.java

```diff
@@ -207,7 +207,8 @@ public enum KeyType {
          * If COLUMN_KEY is MUL, the column is the first column of a nonunique index in which multiple
          * occurrences of a given value are permitted within the column.
          */
-        MUL
+        MUL,
+        AGG
```

Comment:
what dose AGG mean？Could you please add some comments like the other values above?